### DISCUSSION TITLE
Checking - No CLAHE and other image processing for FITS files

### DISCUSSION THREAD

**Discussion by gaitskell** [aaronwmorris/indi-allsky#1221] [Q&A] [answered]
Reference to steps below is for wiki page - Order of Operations: [https://github.com/aaronwmorris/indi-allsky/wiki/Image-order-of-operations]

Do the FITS files images have any processing run on them, or are they effectively raw apart from dark frame subtraction? 
- The wiki doc implies that they are raw-dark, step 2
- Are there any other image processing run on the FITS files that aren't mentioned in the Wiki?

Is the CLAHE run only on the JPG images?  
- The JPG saving Is step 27 on Wiki page?

Just double checking. Thanks, Rick

**Comment by aaronwmorris** [comment, marked as answer]
The only processing done on the data within the FITS files is dark frame subtraction.  Some of the FITS headers might be set or changed, but that is all.  The FITS data is unprocessed otherwise.

Processing, such as CLAHE, would only be performed for the final JPG image.
